src/main/python/rlbot/setup_manager.py

```diff
@@ -45,6 +47,8 @@
 DEFAULT_RLBOT_CONFIG_LOCATION = os.path.realpath('./rlbot.cfg')
 RLBOT_CONFIGURATION_HEADER = 'RLBot Configuration'
 
+TAGAME_CONFIG_LOCATION = os.path.expanduser("~/Documents/My Games/Rocket League/TAGame/Config")
```

Comment:
This path won't work for everyone, for example it's known to be a little different if people have OneDrive enabled, and I could imagine it also being different based on locale. Not a big issue since this is a fringe feature, but I'll be reluctant to advertise it in our wikis etc unless it becomes more robust.

Reply:
Any suggestions to make it more robust?

Reply:
Well, my location is C:\Users\tareh\OneDrive\Documents\My Games\Rocket League, for some stupid reason. Also this works for windows only, obviously.

I wouldn't worry about it much, I think we can just let this feature be mostly undocumented.

Reply:
Do you have any idea of the reason why it is that for you?

Since there used to be a mechanism in RLBot already that edited the .ini files, did it also have to figure out the correct location there? If that is the case, maybe this code can be improved by using the more robust logic to find that folder.

Reply:
It was real simple and real hacky and never supported Linux or Mac:
```
def find_rlbot_ini():
    for games_folder in ['~/Documents/My Games', '~/OneDrive/Documents/My Games']:
        path_to_test = os.path.expanduser(f'{games_folder}/Rocket League/TAGame/Config/TARLBot.ini')
        if os.access(path_to_test, os.F_OK):
            return path_to_test
    return None
```